File: algoritmos-e-logica-de-programacao/Eclipse/ws-eclipse/curso_programacao/src/SomaVetor.java
import java.util.Locale;
import java.util.Scanner;

public class SomaVetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		
		Scanner sc = new Scanner(System.in);
		
		int N;
		double soma = 0, media;
		
		System.out.print("Quantos numeros voce vai digitar? ");
		N = sc.nextInt();
		
		double[] numeros = new double[N];
		
		for (int i = 0; i < N; i++) {
			System.out.print("Digite um numero: ");
			numeros[i] = sc.nextDouble();
		}
		
		System.out.println();
		System.out.print("VALORES = ");
		
		for (int i = 0; i < N; i++) {
			System.out.print(String.format("%.1f ", numeros[i]));
		}
	
		for (int i = 0; i < N; i++) {
			soma = soma + numeros[i];
		}
		
		System.out.println();
		System.out.println("SOMA = " + String.format("%.2f", soma));
		
		media = soma / N;
		
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		sc.close();
	}

}

File: algoritmos-e-logica-de-programacao/PyCharm/ws-pycharm/soma-vetor.py
N = int(input("Quantos numeros voce vai digitar? "))

numeros = [0 for x in range(N)]

soma = 0

for i in range(0, N):
    numeros[i] = float(input("Digite um numero: "))

print()
print("VALORES = ", end="")

for i in range(0, N):
    print(f"{numeros[i]:.1f} ", end="")

print()

for i in range(0, N):
    soma = soma + numeros[i]

print(f"SOMA = {soma:.2f}")

media = soma / N

print(f"MEDIA = {media:.2f}")
